deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +204,67 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payloadEventsString: __ENV.DATA_BTM_SQS_PAYLOAD_EVENTS,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
 }
 
+const payloadEventsArray = JSON.parse(eventData.payloadEventsString)
+
+const payloadTimestampArray = eventData.payloadTimestamp.split(',')
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
+
 const sqs = new SQSClient(awsConfig)
 
-export function sendEvent (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
-  sqs.sendMessage(env.sqs_queue, messageBody)
+export function sendEventType1 (): void {
+  let messageBody: string = ''
+  messageBody = JSON.stringify(payloadEventsArray[0])
+  sendSQSMessage(messageBody)
```

Comment:
Can simplify to a `const` declaration
```suggestion
  const messageBody = JSON.stringify(payloadEventsArray[0])
  sendSQSMessage(messageBody)
```
or avoid the variable declaration entirely
```suggestion
  sendSQSMessage(JSON.stringify(payloadEventsArray[0]))
```